core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -326,7 +326,8 @@ private static boolean isImplicitParsingCast(int fromType, int toType) {
                 || (fromType == STRING && toTag == GEOINT)
                 || (fromType == STRING && toTag == GEOLONG)
                 || (fromType == STRING && toTag == TIMESTAMP)
-                || (fromType == SYMBOL && toTag == TIMESTAMP);
+                || (fromType == SYMBOL && toTag == TIMESTAMP)
```

Comment:
I think it'd be good to add a test showing that conversion now works in insert as select, e.g. 
```sql
create table fromtab ( s string );
create table totab( l long256);
insert into fromtab values ( '0x6bbf0c833a5448baa23a0366d85079afc390e9837e67ac3f653076982d02dd3a' ), ( '6bbf0c833a5448baa23a0366d85079afc390e9837e67ac3f653076982d02dd3a' );
insert into totab select * from fromtab; 
select * from totab ;
```

Reply:
good point added in 5a7b57197e5fbf570edda2c4dbb57e99595cd082